pkg/hash/hash.go

```diff
@@ -50,6 +50,7 @@ func FromString(str string) (Sig, error) {
 		return Sig{}, fmt.Errorf("failed to decode sig '%v': bad len", str)
 	}
 	var sig Sig
+	// nolint:gosimple
```

Comment:
```
pkg/hash/hash.go:53:2: S1001: should use copy() instead of a loop
```
Let's just do instead of the suppression:
```
	copy(sig[:], bin[:])
```

Reply:
Done.